hs_app_netCDF/receivers.py

```diff
@@ -227,21 +232,80 @@ def netcdf_pre_add_files_to_resource(sender, **kwargs):
                     f.resource_file.delete()
                     f.delete()
 
-            # TODO: update resource modification info
-            # user = kwargs['user']
-            # resource_modified(resource, user)
+            # update resource modification info
+            user = kwargs['user']
+            utils.resource_modified(nc_res, user)
 
             # extract metadata
             try:
                 res_dublin_core_meta = nc_meta.get_dublin_core_meta(nc_dataset)
-            except:
+            except Exception:
                 res_dublin_core_meta = {}
 
             try:
                 res_type_specific_meta = nc_meta.get_type_specific_meta(nc_dataset)
-            except:
+            except Exception:
                 res_type_specific_meta = {}
 
+            # update title info
```

Comment:
When I first test on test_metadata_extraction_on_content_file_add() function, it shows fail info.  I found out that in the  netcdf_pre_add_file_to_resource() of receivers.py file, some metadata need to be extracted from netcdf file, which was not implemented. These code are used to extract and update the metadata when calling netcdf_pre_add_file_to_resource(). This will also make the unit test function pass.